### DISCUSSION TITLE
Ingest logs from Kafka

### DISCUSSION THREAD

**Discussion by pfeigl** [datalust/seq-tickets#1542] [Help]
We would like to push our logs into a kafka instance and grab them from there for ingestion into seq.

We saw there is an existing app for RabbitMQ but couldn't find anything that would work with Kafka out of the box.
Other than writing our own `Seq.App` is there any other way to get this setup working?

Thanks,
Philipp

**Comment by liammclennan** [comment]
Hi @pfeigl,

Thank you for opening this discussion. 

I could not find any existing option that can connect kafka to any of Seq's inputs. You could create a separate process that reads kafka and sends to Seq, but that is essentially the same as writing your own Seq app. 

If anyone else needs to connect kafka to Seq please comment on this discussion.
